Formula/confluent-platform.rb

```diff
@@ -11,14 +11,36 @@ class ConfluentPlatform < Formula
 
   conflicts_with "kafka", :because => "kafka also ships with identically named Kafka related executables"
 
+  patch :DATA
+
   def install
     prefix.install "bin"
     rm_rf "#{bin}/windows"
     prefix.install "etc"
     prefix.install "share"
+    prefix.install "libexec"
+    rm_rf "#{libexec}/cli/linux_386"
+    rm_rf "#{libexec}/cli/linux_amd64"
+    rm_rf "#{libexec}/cli/windows_386"
+    rm_rf "#{libexec}/cli/windows_amd64"
   end
 
   test do
     assert_match version.to_s, shell_output("#{bin}/kafka-broker-api-versions --version")
```

Comment:
Could you improve this test to use some of the functionality of the software? Version tests rarely indicate that something is actually wrong. And especially if you're going to patch something we need a test.